### PR TITLE
feat(sampling): Allow to see rule modal without permissions

### PR DESCRIPTION
**Pull request by matejminar**
### Before
Screen Shot 2022-05-25 at 11 45 19

### After
image

### CODE REVIEW

File: static/app/views/settings/project/filtersAndSampling/modal/ruleModal.tsx [hunk 1]
```diff
@@ -303,14 +305,14 @@ function RuleModal({
 
   const predefinedConditionsOptions = conditionCategories.map(([value, label]) => {
     // Never disable the "Add Custom Tag" option, you can add more of those
-    const disabled =
+    const disabledOption =
```
Comment (andriisoldatenko) [inline static/app/views/settings/project/filtersAndSampling/modal/ruleModal.tsx:308]:
```suggestion
    const optionDisabled =
```
just for readability wdyt?

File: static/app/views/settings/project/filtersAndSampling/modal/ruleModal.tsx [hunk 2]
```diff
@@ -411,8 +413,14 @@ function RuleModal({
           <Button
             priority="primary"
             onClick={() => onSubmit({conditions, sampleRate, submitRules})}
-            title={submitDisabled ? t('Required fields must be filled out') : undefined}
-            disabled={isSaving || submitDisabled}
```
Comment (andriisoldatenko) [inline static/app/views/settings/project/filtersAndSampling/modal/ruleModal.tsx:415]: as you can see `Disabled` was here as last part of word.

**Review by andriisoldatenko — CHANGES_REQUESTED** [review]

File: static/app/views/settings/project/filtersAndSampling/modal/ruleModal.tsx [hunk 3]
```diff
@@ -303,14 +305,14 @@ function RuleModal({
...
-    const disabled =
+    const disabledOption =
       value === DynamicSamplingInnerName.EVENT_CUSTOM_TAG
         ? false
         : conditions.some(condition => condition.category === value);
     return {
       value,
       label,
-      disabled,
+      disabled: disabledOption,
```
Comment (andriisoldatenko) [inline static/app/views/settings/project/filtersAndSampling/modal/ruleModal.tsx:315]:
```suggestion
      disabled: optionDisabled,
```

**Review by andriisoldatenko — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]
lgtm

**Review by priscilawebdev — COMMENTED** [review]
should we add unit tests for this change?

**Pull request merged**